docs/reference/esql/functions/ceil.asciidoc

```diff
@@ -0,0 +1,16 @@
+[[esql-ceil]]
+=== `CEIL`
+Round a number up to the nearest integer.
```

Comment:
There should be a comment regarding the data type returned - does CEIL return an int data type or the same data type?
In fact I think we should have a table in the doc similar to PostgresSQL or MsServer.

Reply:
I'm working on making such a table from the docs today! It's something my sick brain can handle....

Reply:
Thanks @nik9000 , your automatic table generation is great.

I'll add the remaining tests so that the other allowed data types (integers and (unsigned) longs) are also documented and will add this and the generated function signature diagram to the docs. Will also update the docs for `FLOOR` so they are aligned.